gradle/libs.versions.toml

```diff
@@ -0,0 +1,103 @@
+[versions]
```

Comment:
@yeikel I believe this answers your question regarding the version catalog

Reply:
It does, but we need to use it more extensively based on my feedback above